tests/units/array/test_indexing.py

```diff
@@ -233,3 +233,18 @@ def test_boolmask_setitem(stack_left, stack_right, da, da_to_set, index):
         else:
             assert d.text == f'hello {i}'
             assert torch.all(d.embedding == torch.ones((4,)) * i)
+
+
+def test_setitiem_update_column():
```

Comment:
does this test ever fail? i think we need to explicitly look at the columns in the da, otherwise it might fool us again like it fooled me

Reply:
let me take a look

Reply:
should I add a test to test that the test fail ???? (kidding ofc)

Reply:
Okay I updated the test, now I check inside the column as well